### PR TITLE
5th round of migration integration tests from Python

### PR DESCRIPTION
**Pull request by robsdedude**

### CODE REVIEW

File: tests/neo4j/sessionrun.py [hunk 1]
```diff
@@ -234,6 +234,25 @@ def test_partial_iteration(self):
...
         self._session1 = None
 
+    def test_simple_query(self):
+        def _test():
+            self._driver.close()
+            self._driver = get_driver(self._backend, userAgent="test")
+            self._session1 = self._driver.session("r", fetchSize=2)
+            result = self._session1.run("UNWIND [1, 2, 3, 4] AS x RETURN x")
+            if consume:
+                result.consume()
```
Comment (fbiville) [inline tests/neo4j/sessionrun.py]: are there assertions we can run on the summary in that case?

File: tests/neo4j/txfuncrun.py [hunk 2]
```diff
@@ -24,6 +24,32 @@ def tearDown(self):
...
 
+    def test_simple_query(self):
+        def _test():
+            self._driver.close()
+            self._driver = get_driver(self._backend, userAgent="test")
+            self._session1 = self._driver.session("r", fetchSize=2)
+            tx = self._session1.beginTransaction()
+            result = tx.run("UNWIND [1, 2, 3, 4] AS x RETURN x")
+            if consume:
+                result.consume()
```
Comment (fbiville) [inline tests/neo4j/txfuncrun.py]: same question as above

File: tests/neo4j/txfuncrun.py [hunk 3]
```diff
@@ -24,6 +24,32 @@ def tearDown(self):
         self._driver.close()
         super().tearDown()
 
+    def test_simple_query(self):
+        def _test():
+            self._driver.close()
+            self._driver = get_driver(self._backend, userAgent="test")
+            self._session1 = self._driver.session("r", fetchSize=2)
+            tx = self._session1.beginTransaction()
```
Comment (fbiville) [inline tests/neo4j/txfuncrun.py]: is this test in the right file? It's testing an explicit tx, while I thought this file was about transaction functions.

Comment (fbiville) [reply tests/neo4j/txfuncrun.py]: I think it's been swapped with the test in `txrun.py` :)

**Review by fbiville — APPROVED** [review]
LGTM, just 1 minor glitch

File: tests/neo4j/txfuncrun.py (hunk 3 above)
Comment (robsdedude) [reply tests/neo4j/txfuncrun.py]: Oh yes. Well spotted, thanks!

File: tests/neo4j/sessionrun.py (hunk 1 above)
Comment (robsdedude) [reply tests/neo4j/sessionrun.py]: We could, but there is `tests.stub.summary.test_summary.TestSummary` which is a whole test suite only for making sure that the summary does what it's supposed to.

Comment (robsdedude) [reply tests/neo4j/sessionrun.py]: :thinking: but a little assertion that roughly the expected thing happened might be a good idea.

**Review by fbiville — APPROVED** [review]

**Pull request merged**